ctapipe_io_lst/__init__.py

```diff
@@ -762,3 +776,31 @@ def __len__(self):
 
     def close(self):
         self.multi_file.close()
+
+    def read_pedestal_ids(self):
+        if self.pedestal_ids_path is not None:
+            t = read_table(self.pedestal_ids_path, '/interleaved_pedestal_ids')
+            Provenance().add_input_file(
+                self.pedestal_ids_path, role="InterleavedPedestalIDs"
+            )
+            self.pedestal_ids = set(t['event_id'])
+        else:
```

Comment:
shouldn't we throw here a warning that no `pedestal_ids_path` was given?

Reply:
No, I don't think so, since giving it is only required for a small number of old runs that don't properly tag the events, If I understood the problem correctly.